Methane_MarkDown.Rmd

```diff
@@ -111,3 +109,46 @@ ggplot(data= results, aes(x= year, y= value, color= units))+
          color = "Units") + facet_wrap(~variable)
 ```
 ... to show our data side-by-side.
+
+## Sensitivity Analysis
+
+To examine SSP245's Natural methane  sensitivity we first ask hector to fetch the data. We then create a function to run our parameter set to a particular value. In this case we set our value at `358.05`- twenty percent higher than the original value.
+
+```{r run-with-parameters}
+core <- newcore(ini_file)
+run(core)
+paramr<- fetchvars(core, NA, NATURAL_CH4())
+param<- function(core, parameter, value) {
+  old_value <- fetchvars(core, NA, parameter)
+  unit <- as.character(old_value[["units"]])
+  setvar(core, NA, parameter, value, unit)
+  reset(core)
+  run(core)
+  result <- fetchvars(core, 2000:2100)
+  result[["parameter_value"]] <- value
+  result
+}
+```
+
+Then we tell hector to run a range of values. In this case we ask for 20% above and bellow the set value of `NATURAL_CH4`(341Tg) at intervals of 5. 
+
+```{r sen-param}
+run_with_param_range <- function(core, parameter, values) {
+  mapped <- Map(function(x) param(core, parameter, x), values)
+  Reduce(rbind, mapped)
+}
+sensitivity_emission <- run_with_param_range(core, NATURAL_CH4(), seq(272.8, 358.05, 5))
+```
+
+Once we have our differing parameter values we can create a graph. Using the graph function `facet_wrap`to separate the results based on variable. 
+
+```{r sen-graph}
+ggplot(sensitivity_emission) +
+  aes(x = year, y = value, color = parameter_value, group = parameter_value) +
```

Comment:
Nice graph! Maybe add a `linetype = units` or `shape = units` or something in your `aes()` call to have the units appear in the legend,